ivy/compiler/compiler.py

```diff
@@ -32,16 +32,16 @@ def compile(
     kwargs: Optional[dict] = None,
 ) -> Union[Graph, LazyGraph]:
     if python_version[1] == "8":
-        from ._compiler_38 import compile as _compile
+        from ._compiler_38 import trace as _trace
     else:
-        from ._compiler import compile as _compile
+        from ._compiler import trace as _trace
```

Comment:
small heads up: this will need to be changed to `from ._compiler import compile as _trace` as the .so won't be updated yet

Reply:
yep!